pkg/api/router.go

```diff
@@ -81,6 +81,12 @@ func Run(apiServerCfg cfg.OsdsApiServer) {
 			beego.NSRouter("/:tenantId/pools/:poolId", &PoolPortal{}, "get:GetPool"),
 			beego.NSRouter("/:tenantId/availabilityZones", &PoolPortal{}, "get:ListAvailabilityZones"),
 
+                        // Share is a part of files. At the same time multiple users can access the the same shares.
```

Comment:
This should now be moved into a file.go file similar to what has been done with the block endpoint routing.

https://github.com/opensds/opensds/blob/4a4f66699d0019b6f150bbb902e9ac2828c47b86/pkg/api/routers/block.go

Reply:
@stmcginnis Who did all these file changes, there are lot of duplication of code? May i know the purpose ?

Reply:
@stmcginnis sorry there are no duplication of code, only file structure has been changed to separate out for block and fileshare